providers/angus-mail/src/test/java/org/eclipse/angus/mail/util/SocketFetcherTest.java

```diff
@@ -534,71 +534,27 @@ public void testSSLCheckServerIdentityFalse() throws Throwable {
     }
 
     @Test
-    public void testSSLCheckServerIdentityNull() {
-        try {
-            testSSLCheckServerIdentity("localhost", (String) null);
-            throw new AssertionError();
-        } catch (Error | RuntimeException e) {
-            throw e;
-        } catch (MessagingException me) {
-            Throwable cause = me.getCause();
-            assertTrue(String.valueOf(cause),
-                    cause instanceof SSLHandshakeException);
-            assertTrue(me.toString(), isFromTrustManager(me));
-        } catch (Throwable t) {
-            throw new AssertionError(t);
-        }
+    public void testSSLCheckServerIdentityNull() throws Throwable {
+        // localhost should fail? See TestSSLSocketFactory#84 -> ((MailSSLSocketFactory) defaultFactory).setTrustedHosts("localhost");
```

Comment:
The point of the test was assert that the EIA was being enforced by the JDK.  I think only change needed for original code is to switch paramter from "localhost" to "mailtest.local" in each test.